cluster/engine.go

```diff
@@ -192,13 +192,13 @@ func (e *Engine) Connect(config *tls.Config) error {
 	e.httpClient = httpClient
 	e.url = url
 
-	// Use HTTP Client created above to create a dockerclient client
+	// Use HTTP Client created above to create a docker client
```

Comment:
`dockerclient` is referring to `github.com/samalba/dockerclient`. It's not typo.